pyresample/geometry.py

```diff
@@ -856,7 +856,7 @@ def __eq__(self, other):
         """Test for equality"""
 
         try:
-            return ((self.proj_dict == other.proj_dict) and
+            return ((self.proj_str == other.proj_str) and
```

Comment:
not sure about this one, since `"+proj=geos +lon_0=0"` is different from `"+lon_0=0 +proj=geos"` as a string but not as a dict

Reply:
The proj_str property sorts the string so it is the same. The only issue would be something like one area specifying a parameter that is a default (`lon_0=0` is the same as not including it).

Reply:
The `proj_dict` is not hashable so I couldn't use it in the hash method.